### DISCUSSION TITLE
Add window title bar color property

### DISCUSSION THREAD

**Discussion by ghost** [flet-dev/flet#813] [Ideas]
it would be nice if the window title bar for desktop app mode could be colored with custom color and/or detect system theme and use those colors.

Im working on a desktop app which runs in dark mode, but the window title bar of flet.exe stays bright, which looks kinda weird. Hiding it and building my own title bar seems to be the only workaround right now.

**Comment by FeodorFitsner** [comment]
The color of window title bar is controlled by an OS current theme as far as I understand? Yes, you can build your own title bar, take a look at this example: https://github.com/flet-dev/examples/blob/main/python/controls/window-drag-area/no-frame-window.py

**Reply by ghost** [reply]
on two machines running win11 home the os theme does not style the flet.exe window title but every other processes windows. I'll try to investigate further tomorrow. Maybe the problem relies on my side, but im still suprised it's only the window title that is not affected by dark theme while everything inside the page is in dark mode with os highlight colors and so on.

**Comment by ghost** [comment]
I can reproduce the behavior/cannot fix it. 

I started a new venv with python 3.10 on window running flet 0.3.2. with some example code from the tutorials:
(same behaviour if i pack it with "flet pack" or pyinstaller)
```
import flet as ft

def main(page: ft.Page):
    def add_clicked(e):
        page.add(ft.Checkbox(label=new_task.value))
        new_task.value = ""
        page.update()

    new_task = ft.TextField(hint_text="Whats needs to be done?")

    page.add(new_task, ft.FloatingActionButton(icon=ft.icons.ADD, on_click=add_clicked))

ft.app(target=main)
```
it looks like this. only flet is affected by this behavior:
![Screenshot 2023-01-07 121007](https://user-images.githubusercontent.com/121888212/211147203-3eed2158-e432-43c0-81d8-3b1a13bc0003.png)

my windows settings are set to dark mode.:
![Screenshot 2023-01-07 121334](https://user-images.githubusercontent.com/121888212/211147383-5ea4f13c-1454-495c-b627-41a3c9cb50ef.png)

Any ideas?

**Reply by ghost** [reply]
reminds me of the same problem with signal desktop app in win11 (despite issue sayin win10) https://github.com/signalapp/Signal-Desktop/issues/2696

**Comment by Pegioner** [comment]
I was looking for this property in Flet and found this discussion.
This is actually a useful feature for a cross platform application. To be able to stay in the style of the OS, which will not work with a custom title bar (because, for example, the title bar in Windows and MacOS has different heights, styles). A white title bar with a dark theme looks some ugly.
